audiosource

```diff
@@ -10,9 +10,7 @@ if [ "$AUDIOSOURCE_PROFILE" = 'release' ]; then
 fi
 
 AUDIOSOURCE_APK=${AUDIOSOURCE_APK:-"$AUDIOSOURCE_DEFAULT_APK"}
-AUDIOSOURCE_NAME=${AUDIOSOURCE_NAME:-android}
-AUDIOSOURCE_PIPE=${AUDIOSOURCE_PIPE:-/tmp/audiosource}
-AUDIOSOURCE_SOCKET=${AUDIOSOURCE_SOCKET:-audiosource}
+AUDIOSOURCE_NAME=${AUDIOSOURCE_NAME:-audiosource$(uuidgen)}
```

Comment:
The name should be stable, because if you adjust the volume or other settings in pavucontrol, they would be lost if you launch it again. One way of doing that would be deriving a short identifier from the sha256 sum of the serial (e.g. `audiosource-82ab264`).